Assets/Resources/Microgames/KaguyaMemory/Scripts/KaguyaMemory_RNGDeciderScript.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KaguyaMemory_RNGDeciderScript : MonoBehaviour {
+
+    [SerializeField]
+    private AudioClip whooshSound;
+
+    public bool finished = false;
+    public int maxItems = 3;
+    private int randomNumber = 0;
+    public GameObject item1;
+    public GameObject item2;
+    public GameObject item3;
+    public GameObject item4;
+    public GameObject item5;
```

Comment:
There's no need to have 5 variables here, just use a list (public lists will still show up in the inspector)